main.go

```diff
@@ -135,8 +135,8 @@ func initSelfNodeRemediationManager(mgr manager.Manager) {
 	setupLog.Info("Starting as a manager that installs the daemonset")
 
 	if err := utils.InitOutOfServiceTaintSupportedFlag(mgr.GetConfig()); err != nil {
-		setupLog.Error(err, "unable to verify out of service taint support")
-		os.Exit(1)
+		utils.IsOutOfServiceTaintSupported = false
```

Comment:
This is redundant, it's the default value. (and even if it wouldn't be the default value, setting that global var, which isn't nice on its own already, here and not in its own package, wouldn't be nice as well IMHO)

Reply:
True, I think it's a bit more readable this way (i.e an error occurred so we are explicitly disabling this feature) but it's definitely not something I'm feeling strongly about, so I don't mind removing it.